docs/specifications/deep-links.md

```diff
@@ -0,0 +1,190 @@
+applications rather than a website in a browser.
+They are useful for a number of things, however in our case
+we are interested in the user experiences that they enable
+between websites, applications, platforms, etc. by providing more interoperability.
+
+Firefly has its own deep link scheme, exposing (limited) functionality that is required in 
+some type of user flow. A trivial example would be a user is buying native tokens on Soonaverse and must make a payment transaction
+in order to execute the buy order. Clicking on a deep link embedded inside the Soonaverse platform will trigger Firefly to 
+open and auto-fill the transaction data as necessary, making it simple confirm and click job for the user. 
+
+:::caution
+Firefly **will NEVER** automatically execute actions initiated by a deep link; they should **ALWAYS** require manual 
+confirmation on behalf of the user.
+:::
+
+## Scheme
+
+The Firefly deep link scheme can be broken down to the following (simple) syntax:
+
+```
+firefly[-<stage>]://<context>/<operation>[?param=<param>]
+```
+
+The parameters are as follows:
+
+- `stage` - indicates a specific stage of the app to target, options are:
+  - `alpha` - the first available version of Firefly containing brand new features
+  - `beta` - the next available version of Firefly containing new but slightly tested features
+  - `shimmer` - the Firefly Shimmer version, containing new and well-tested features 
+- `context` - the part of Firefly that contains the operation, options are:
+  - `wallet` - managing coins and tokens
+  - `collectibles` - managing NFTs
+  - `governance` - managing voting events and proposals
+- `operation` - an operation within a specific context (see below for more detail)
+- `param` - query parameter(s) relevant for the specified operation
+
+If you wish to target the production version, simply omit this from the prefix:
+
+```
+firefly://
+```
+
+:::caution
+This deep link scheme is **NOT** compatible with Firefly V1, as that version of the application is in maintenance mode.
+:::
+
+## Contexts
+
+### Wallet
+
+#### Send Form
+
+This operation brings the user to the send form popup:
+
+:::image
+![](../static/send-form-popup.png "Send form popup")
+:::
+
+The deep link structure is as follows:
+
+```
+firefly://wallet/sendForm?address=<address>&amount=<amount>[&unit=<unit>][&assetId=<assetId>][&metadata=<metadata>][&tag=<tag>]
+```
+
+The following parameters are **required**:
+
+- `address` - the recipient's address where the funds will be sent to
+  - **MUST** be a Bech32 address; considering support for other address types in the future
+- `amount` - the amount of tokens to send in the transaction
+  - **MAY** contain a decimal so long as it makes sense given the value of the `unit` param (see below)
+
+The following parameters are **optional**:
+
+- `unit` - a specified denomination of the token to use, if applicable (default for IOTA is `Mi`, SMR is `SMR`)
```

Comment:
And then we can delete this. I have a strong opinion on this, but I might be wrong. So if you feel strongly opposed, feel free to convince me :D